### ISSUE TITLE
Sync production version of BFN with Github and create a release

### ISSUE THREAD

**Issue by wunluv** [ContextInstitute/bfcom#13]
It appears that bfn.context.org is not in sync with Github. Its useful to have a master -> release flow such that a released version indicated by a tag on github is always used in production. The master branch is used to create the next release.

With the recent upgrade of WP, the master branch should reflect the upgrade changes.

**Comment by contextinst** [comment]
Agreed. As I work my way back to active development, I plan to get everything in sync and then follow the process in [GitHub Workflow](https://github.com/ContextInstitute/bfcom/wiki/Github-Workflow). I am also planning on installing the WP plugin [GitHub Updater](https://github.com/afragen/github-updater/wiki) to automate the process of keeping the DO dev site synced with the GH master.
